app/controllers/menus_controller.rb

```diff
@@ -0,0 +1,46 @@
+class MenusController < ApplicationController
+  def index
+    authorize Menu
+    @menus = Menu.all
+  end
+
+  def new
+    authorize Menu
+    @facade = Menus::NewFacade.new
+
+    redirect_weekend
+
+    redirect_menu_exist
+  end
+
+  def create
+    @menu = Menu.new(menus_params)
+
+    redirect_to menus_path and return if @menu.save
+
+    render :new
+  end
+
+  private
+
+  def menus_params
+    params
+      .require(:menu)
+      .permit(:name,
+              :price,
+              :date,
+              meals_attributes: %i[price food_item_id _destroy])
+  end
+
+  def redirect_weekend
+    redirect_to root_path, flash: {
```

Comment:
[redirect_to doesn't stop the execution of the function](http://api.rubyonrails.org/classes/ActionController/Redirecting.html)

You need to add `return` to `redirect`